test/org/sagebionetworks/bridge/services/SurveyServiceMockTest.java

```diff
@@ -627,7 +614,6 @@ public void deleteSurveyPermanentlyInOtherStudyThrowsException() {
     public void getSurveyMostRecentlyPublishedVersionInOtherStudy() {
         Survey survey = Survey.create();
         survey.setStudyIdentifier(TestConstants.TEST_STUDY_IDENTIFIER);
-        when(mockSurveyDao.getSurveyMostRecentlyPublishedVersion(TestConstants.TEST_STUDY, SURVEY_GUID, true)).thenReturn(survey);
```

Comment:
I've fixed these two tests so they test what they are supposed to be testing, however, as you note, the DAO should never return a survey with a study ID that's different than the study ID you supply to the DAO, because the study ID is part of the DDB query. Filed BRIDGE-2366 to remove these checks entirely (the comment about this references admins but doesn't make any sense to me... I can't think of any reason to enforce this again in the service).